src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+
+import { App } from './App';
+
+ReactDOM.render(
+  <BrowserRouter>
```

Comment:
use HashRouter instead of BrowserRouter
https://wanago.io/2021/04/19/hashrouter-browserrouter-react/

because now when I reload a page I see 404 not found and I should see the same content after reload
> After the page reload you should see a content of the tab matching the id from the URL